plugins/catalog/src/components/AboutCard/AboutCard.tsx

```diff
@@ -109,7 +117,12 @@ export function AboutCard({ entity, variant }: AboutCardProps) {
       <CardHeader
         title="About"
         action={
-          <IconButton href={codeLink.href || '#'} aria-label="Edit">
+          <IconButton
+            aria-label="Edit"
+            onClick={() => {
+              window.open(codeLink.edithref || '#', '_blank');
```

Comment:
disagree with this change - we should in general not open new windows forcibly unless necessary - the user can choose which they prefer